concent_api/core/utils.py

```diff
@@ -118,3 +121,27 @@ def extract_name_from_scene_file_path(absoulte_scene_file_path_in_docker: str) -
             ErrorCode.MESSAGE_INVALID
         )
     return relative_scene_file_path_in_archive
+
+
+def is_protocol_version_compatible(protocol_version: str) -> bool:
+    """
+    Versions are considered compatible if they share the minor and major version number. E.g 2.18.5 is compatible with
+    2.18.1 but not with 2.17.5 or 3.0.0. This supports semver version style https://semver.org/
+    """
+    return protocol_version.split('.')[0] == settings.GOLEM_MESSAGES_VERSION.split('.')[0] and \
+        protocol_version.split('.')[1] == settings.GOLEM_MESSAGES_VERSION.split('.')[1]
+
+
+def is_given_golem_messages_version_supported_by_concent(
```

Comment:
are there tests for that?

Reply:
yes, in `TestValidateCompatibilityGolemMessages`